scripts/targeted-documentation.js

```
@@ -67,13 +67,13 @@ const keyFiles = [
     'apps/web-giddh/src/app/login/login.component.ts',
     'apps/web-giddh/src/app/contact/contact.component.ts'
 ];
-let totalAdditions = 0;
-let filesProcessed = 0;
+const totalAdditions = 0;
+const filesProcessed = 0;
 keyFiles.forEach(filePath => {
     const fullPath = path.join(__dirname, '..', filePath);
     if (fs.existsSync(fullPath)) {
         const additions = addDocumentationToFile(fullPath);
         totalAdditions += additions;
```

Biome (2.1.2)

[error] Can't assign totalAdditions because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)